src/main/java/team/TeamLeader.java

```diff
@@ -0,0 +1,76 @@
+package team;
+
+import equipment.Equipment;
+import exceptions.ItemNotFoundException;
+import exceptions.MemberNotFoundException;
+
+import java.util.HashMap;
+
+public class TeamLeader extends TeamMember {
+    private HashMap<String, TeamMember> team;
+
+    public TeamLeader(String personalNumber) {
+        super(personalNumber);
+        this.team = new HashMap<String, TeamMember>();
+    }
+
+    public TeamLeader(String personalNumber, HashMap<String, TeamMember> team) {
+        super(personalNumber);
+        this.team = team;
+    }
+
+    public void addTeamMember(TeamMember teamMember) {
+        team.put(teamMember.getPersonalNumber(), teamMember);
+    }
+
+    public TeamMember removeTeamMember(String personalNumber) {
+        return team.remove(personalNumber);
+    }
+
+    /**
+     * Method to return combined inventory of all team members
+     *
+     * @return HashMap of equipment ID keys pointing to matching equipment objects
+     */
+    public HashMap<String, Equipment> getTeamInventory() {
```

Comment:
OCP- adding new types of equipment summery will require more and more changes to the `TeamLeader`, make the summarizers separate classes